src/test/integration/suite/environments.test.it.ts

```diff
@@ -0,0 +1,98 @@
+import * as assert from "assert";
+import { mockGetEnvironment, mockGetOrganization } from "../mocks/server";
+import { mockGitRepoAndBranch } from "../mocks/git";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import * as extension from "../../../../dist/extension.js";
```

Comment:
this is import of the extension main file that runs by vscode.
Importing that in order to be able to communicate with the extension.